doc/manual/parallel-computing.rst

```diff
@@ -113,15 +113,15 @@ Note that we used ``1 .+ fetch(r)`` instead of ``1 .+ r``. This is because we
 do not know where the code will run, so in general a :func:`fetch` might be
 required to move ``r`` to the process doing the addition. In this
 case, :obj:`@spawn` is smart enough to perform the computation on the
-process that owns ``r``, so the :func:`fetch` will be a no-op.
+process that owns ``r``, so the :func:`fetch` will be a no-op (no work is done).
 
 (It is worth noting that :obj:`@spawn` is not built-in but defined in Julia
 as a :ref:`macro <man-macros>`. It is possible to define your
 own such constructs.)
 
-An important thing to remember is that, once fetched, a ``Future`` will cache its value
-locally. Further ``fetch`` calls do not entail a network hop. Once all referencing Futures
-have fetched, the remote stored value is deleted.
+An important thing to remember is that, once fetched, a :class:`Future` will cache its value
+locally. Further :func:`fetch` calls do not entail a network hop. Once all referencing
+:class:`Future`s have fetched, the remote stored value is deleted.
```

Comment:
needs a space before the s `doc/manual/parallel-computing.rst:122: WARNING: Inline interpreted text or phrase reference start-string without end-string.`